src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -3,87 +3,81 @@
 
 # NB: The project names must follow the naming scheme at
 #  https://www.python.org/dev/peps/pep-0503/#normalized-names.
+
+import re
+from functools import partial
+from typing import Match
+
+
+def all_hyphen_to_dot(m: Match) -> str:
```

Comment:
**comment:** functions for reusability?
tried to be very verbose with the names so these can easily be used in the future for other pattern matches.
opted to not use `lambdas`, because they are anonymous, or simple "string" repls because i thought they'd be less likely to be reused in the future.